app/assets/stylesheets/earthworks.css

```diff
@@ -152,3 +152,37 @@ Uncomment lines 142-144
     display: none;
   }
 }
+
+.filter .btn-outline-secondary {
+  --bs-btn-border-color: var(--stanford-fog-light);
+  --bs-btn-color: var(--stanford-black);
+  background-color: var(--stanford-fog-light);
+  --bs-btn-bg-color: var(--stanford-fog-light);
+  --bs-btn-hover-color: var(--stanford-black);
+  --bs-btn-hover-border-color: var(--stanford-fog-light);
+  --bs-btn-active-border-color: var(--stanford-fog-light);
+  --bs-btn-active-color: var(--stanford-black);
+  --bs-btn-padding-y: .25rem;
+}
+
+.applied-filter .remove:hover {
+  --bs-btn-hover-border-color: var(--stanford-fog-light);
+  --bs-btn-hover-bg: var(--stanford-fog-light);
+  --bs-btn-hover-color: var(--stanford-digital-red);
+  text-decoration: none;
+}
+
+.filter .constraint-value {
+  border-radius: 99px 0px 0px 99px;
+  padding-right: .25rem;
+}
+
+.filter .remove {
```

Comment:
Aren't `.filter` and `.applied-filter` on the same elements?  Why is this one separate?

Reply:
good catch. They should all be applied-filter. I noticed later in the process that filter isn't applied to queries, but applied-filter is. I have updated all the css to that.